app/controllers/users_controller.rb

```diff
@@ -20,7 +20,7 @@ def tree
       @title << " By Karma"
       render :action => "list"
     elsif params[:moderators]
-      @users = User.where("is_admin = 1 OR is_moderator = 1").
+      @users = User.where("is_admin = true OR is_moderator = true").
```

Comment:
`where("is_admin = ? OR is_moderator = ?", true, true)` would probably be best, so AR can figure out what is best